readme.md

```diff
@@ -41,9 +41,18 @@ Type: `object`
 ##### https
 
 Type: `boolean`<br>
-Default: `false`
+Default: `null`
 
-Use a HTTPS check using the [icanhazip.com](https://github.com/major/icanhaz) service instead of the DNS query. [ipify.org](https://www.ipify.org) is used as a fallback if `icanhazip.com` fails. This check is much more secure and tamper-proof, but also a lot slower. **This option is only available in the Node.js version**.
+Use a HTTPS check using the [icanhazip.com](https://github.com/major/icanhaz) service instead of the DNS query. [ipify.org](https://www.ipify.org) is used as a fallback if `icanhazip.com` fails. This check is much more secure and tamper-proof, but also a lot slower. **This option is only available in the Node.js version**. Default behaviour is to check aginst DNS before using HTTPS fallback, if set as `true` it will *only* check against HTTPS, and if set to `false` it will *only* check agains DNS.
+
+##### urls
```

Comment:
Maybe we should call it `fallbackUrls` to make its purpose clear?

Reply:
changed to use `fallbackUrls `